db/migrate/20201125151816_create_schools.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSchools < ActiveRecord::Migration[6.0]
+  def change
+    create_table :schools do |t|
+      t.timestamps
+      t.column :urn, :string, null: false
+      t.column :name, :string, null: false
+      t.column :address, :string, null: false
+      t.column :school_type, :string
+      t.column :capacity, :integer
+      t.column :high_pupil_premium, :boolean, null: false, default: false
+      t.column :is_rural, :boolean, null: false, default: false
+    end
+    add_index :schools, :urn, unique: true
+    add_index :schools, :name
+    add_index :schools, :high_pupil_premium, where: :high_pupil_premium
```

Comment:
I added these partial indices since I assume we're only interested in finding schools where this is true. Not particularly confident about this in general, so would be great to hear someone else's view

Reply:
Sounds like a great idea! 👍